apis/management.cattle.io/v3/schema/schema.go

```diff
@@ -66,7 +66,15 @@ func clusterTypes(schemas *types.Schemas) *types.Schemas {
 			schema.SubContext = "clusters"
 		}).
 		MustImport(&Version, v3.ClusterEvent{}).
-		MustImport(&Version, v3.ClusterRegistrationToken{})
+		MustImport(&Version, v3.ClusterRegistrationToken{}).
+		MustImportAndCustomize(&Version, v3.Cluster{}, func(schema *types.Schema) {
+			schema.MustCustomizeField("name", func(field types.Field) types.Field {
+				field.Type = "dnsLabel"
```

Comment:
any chance `dnsLabel` is a constant in norman? so we can use that constant instead of hardcoding the naming